api/v1alpha1/openapi.yaml

```diff
@@ -1743,6 +1976,7 @@ components:
       - DeviceAvailable
       - DeviceProgressing
       - DeviceDegraded
+      - TemplateVersionReady
```

Comment:
I am not openapi expert so i am curious what is the best way to document these. For example it would be nice to understand what `TemplateVersionReady` means.

Reply:
I suppose we can add # comments?